packages/tools/ds-version-history/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
rename the folder to `figma-plugin-version-history` since we'll be prefixing all the plugins with this so it'll be organised in a nice manner in the tree hierarchy in the directory structure

Reply:
Done @kamleshchandnani!